tests/test-amp-analytics-options.php

```diff
@@ -75,8 +88,9 @@ function test_no_options() {
 	 * Test that exactly one analytics component is inserted into the DB
 	 */
 	function test_one_option_inserted() {
-		/* Delete analytics options, if any */
-		delete_option( 'analytics' );
+
+		// Delete options, if any
+		delete_option( 'amp-options' );
```

Comment:
Related to the ROLLBACK comment before: some tests were failing seemingly due the DB not being rolled back between tests; it should be but... Adding this delete_option for now.